pyproject.toml

```diff
@@ -35,6 +35,7 @@ dev = [
 ]
 test = [
     "pytest",
+    "requests-mock"
```

Comment:
Interesting - I've always used [response](https://pypi.org/project/response/) for this, didn't know there were multiples.

You likely don't care about async stuff, but httpx & respx have also treated me well so far.

Reply:
yeah - requests-mock looks like it's a bit more actively maintained, but i've used response in the past.

for the purposes of this, i don't think async makes sense?